app/serializers/rest/status_serializer.rb

```diff
@@ -3,7 +3,8 @@
 class REST::StatusSerializer < ActiveModel::Serializer
   attributes :id, :created_at, :in_reply_to_id, :in_reply_to_account_id,
              :sensitive, :spoiler_text, :visibility, :language,
-             :uri, :content, :url, :reblogs_count, :favourites_count
+             :uri, :content, :url, :reblogs_count, :favourites_count,
+             :license, :license_url
```

Comment:
It feels like it would be a bit cleaner, API wise, to have a `license: { name: ..., url: ... }` object